src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborWriterTests.String.cs

```diff
@@ -72,12 +72,12 @@ public static void WriteTextString_IndefiniteLength_SingleValue_HappyPath(string
         }
 
         [Fact]
-        public static void WriteTextString_InvalidUnicodeString_ShouldThrowEncoderFallbackException()
+        public static void WriteTextString_InvalidUnicodeString_ShouldThrowArgumentException()
         {
             // NB Xunit's InlineDataAttribute will corrupt string literals containing invalid unicode
             string invalidUnicodeString = "\ud800";
             using var writer = new CborWriter();
-            Assert.Throws<System.Text.EncoderFallbackException>(() => writer.WriteTextString(invalidUnicodeString));
+            Assert.Throws<ArgumentException>(() => writer.WriteTextString(invalidUnicodeString));
```

Comment:
```suggestion
            ArgumentException ex = AssertExtensions.Throws<ArgumentException>(
                "whateverParameterName",
                () => writer.WriteTextString(invalidUnicodeString));
            Assert.NotNull(ex.InnerException);
            Assert.IsType<System.Text.EncoderFallbackException>(ex.InnerException);
```

The code is off the top of my head, but should be close to the right answer.

Reply:
Fixed.